Rubberduck.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2010
+# Visual Studio Version 16
```

Comment:
Looks like this PR drives a move to VS 16 and net461 (see `RubberduckBaseProject.csproj` below).  True?

Reply:
Yes required for the `System.Abstractions.IO` nuget package.